src/index.ts

```diff
@@ -290,3 +291,50 @@ function scenarioNotAuthorised(
     authenticated: false
   }
 }
+
+
+
+// ㊙️
+
+
+async function importClassifiedInfo(
+  classifiedParam: Maybe<string>
+): Promise<void> {
+  if (!classifiedParam) return
+
+  const ks = await keystore.get()
+  const classifiedInfo = JSON.parse(common.base64.urlDecode(classifiedParam))
+
+  // Extract session key and its iv
+  const iv = utils.base64ToArrBuf(classifiedInfo.iv)
+  const rawSessionKey = await ks.decrypt(classifiedInfo.sessionKey)
+  const sessionKey = await crypto.subtle.importKey(
```

Comment:
can also use `keystore.aes.importKey` if ya like

Reply:
Ah nice, thx, will look at that.